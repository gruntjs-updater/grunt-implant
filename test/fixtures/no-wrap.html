<!DOCTYPE html>
<html>
<head>
</head>
<body>
<!--implant:no-wrap-->
</body>
</html>