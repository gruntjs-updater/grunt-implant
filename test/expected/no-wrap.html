<!DOCTYPE html>
<html>
<head>
</head>
<body>
ABC
</body>
</html>